### PR TITLE
use path gen functions from ps-pathy

### PR DESCRIPTION
**Pull request by safareli**

### CODE REVIEW

File: src/Quasar/Mount/Common/Gen.purs [hunk 1]
```diff
@@ -67,17 +66,10 @@ genCredentials =
...
-    parts ∷ L.List String ← Gen.unfoldable genAlphaNumericString
-    pure $ foldr (flip P.appendPath <<< P.dir) P.rootDir parts
+genDirPath = PGen.genAbsDirPath
 
 genFilePath ∷ ∀ m. MonadGen m ⇒ MonadRec m ⇒ m FilePath
-genFilePath = do
-  dir ← genDirPath
-  file ← genAlphaNumericString
-  pure $ dir </> P.file file
+genFilePath = PGen.genAbsFilePath
```
Comment (cryogenian) [inline src/Quasar/Mount/Common/Gen.purs]: Why do we need this be redefined here? You can just reexport them and rename, right? Anyways that's not public API

Comment (safareli) [reply src/Quasar/Mount/Common/Gen.purs]: kk will rename

**Comment by safareli** [comment]
@cryogenian  renamed

**Review by cryogenian — APPROVED** [review]
:+1:

**Pull request merged**